tracing-futures/src/lib.rs

```diff
@@ -381,6 +381,18 @@ impl<T> Instrumented<T> {
         &mut self.inner
     }
 
+    /// Get a pinned reference to the wrapped type.
+    #[cfg(feature = "std-future")]
```

Comment:
Can we also add 
```rust
#[cfg_attr(docsrs, doc(cfg(feature = "std-future")))]
```
attributes to all the new feature-flagged methods added in this PR? That way, the RustDoc for the crate will show that the `std-future` feature flag is required to enable these methods.

Thanks!

Reply:
oh gosh, I hadn't payed attention to those, and here I was so happy thinking that rustdoc now automatically picks up on features. I'll have to add a lot to my own crates. 

I will add those.